src/libecalc/presentation/exporter/dto/dtos.py

```diff
@@ -100,6 +116,18 @@ def check_equal_lengths_of_timeseries(cls, values):
 
         return values
 
+    def fit_to_timesteps(self, timesteps: List[datetime]) -> "TSVPrognosis":
+        return TSVPrognosis(
+            time_steps=TimeSteps(
+                values=timesteps,
+                frequency=self.time_steps.frequency,  # Should this be provided?
```

Comment:
What's the frequency here @TeeeJay? Just output frequency used for the run? Is it needed?

Reply:
https://en.wikipedia.org/wiki/What%27s_the_Frequency,_Kenneth%3F

Reply:
nice

Reply:
but yes, I think it is just "metainfo" , but possibly used to know what to expect from the timeseries interval. shouldnt be needed now i think, as we can change to whatever we want, irrelevant of original frequency